docs/source/notebooks/DEMetropolisZ_tune_drop_fraction.ipynb

```diff
@@ -0,0 +1,1795 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DEMetropolis(Z): tune_drop_fraction\n",
+    "The implementation of `DEMetropolisZ` in PyMC3 uses a different tuning scheme than described by [ter Braak & Vrugt, 2008](https://doi.org/10.1007/s11222-008-9104-9).\n",
+    "In our tuning scheme, the first `tune_drop_fraction * 100` % of the history from the tuning phase is dropped when the tune iterations end and sampling begins.\n",
+    "\n",
+    "In this notebook, a D-dimenstional multivariate normal target densities is sampled with `DEMetropolisZ` at different `tune_drop_fraction` settings to show why the setting was introduced."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy\n",
+    "import pandas\n",
+    "from matplotlib import pyplot\n",
+    "from matplotlib import gridspec\n",
+    "from matplotlib import cm\n",
+    "import ipywidgets\n",
+    "import time\n",
+    "import pymc3\n",
+    "import arviz\n",
+    "import theano.tensor as tt"
```

Comment:
I think this import is not used. Also could we follow the convention of using alias when importing the libraries like `import numpy as np`, I think this is the style we follow in most notebooks.

Reply:
I can do that. However I really don't like this "convention" of aliasing a 5-letter package into a 2-letter alias. I've seen many beginners being super confused by this.